resources/src/androidMain/kotlin/dev/icerock/moko/resources/desc/StringDesc.kt

```diff
@@ -18,9 +18,9 @@ actual interface StringDesc {
         }
 
         actual class Custom actual constructor(
-            locale: String
+            languageTag: String
         ) : LocaleType() {
-            override val systemLocale: Locale = Locale(locale)
+            override val systemLocale: Locale = Locale.forLanguageTag(languageTag)
```

Comment:
lint failed here. `forLanguageTag` supported only from 21 API. for old api we should have another implementation

Reply:
we can build `Locale` object from lang, country and variant, just like in Apple version implemented now

Reply:
Nice, good catch. Replaced in 0b0db64eec96760eae5762d82d657e530eb07ac3